packages/lodestar-beacon-state-transition/src/fast/util/persistentVector.ts

```diff
@@ -0,0 +1,332 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const BIT_WIDTH = 5;
+const BIT_MASK = 0b11111;
+const BRANCH_SIZE = 1 << BIT_WIDTH;
+const DEFAULT_LEVEL_SHIFT = 5;
+
+function isFullBranch(length: number): boolean {
+  return (
+    // initially we initialize Vector with an empty branch (DEFAULT_LEVEL_SHIFT)
+    // length === 1 << 5 ||
+    length === 1 << 10 || length === 1 << 15 || length === 1 << 20 || length === 1 << 25 || length === 1 << 30
+  );
+}
+
+interface VLeaf<T> {
```

Comment:
What's up with V prefix. Aren't we using I everywhere?

Reply:
V may stands for `Vector`, that's the upstream implementation, will fix it